examples/qml_multi_crates/rust/main/src/lib.rs

```diff
@@ -0,0 +1,12 @@
+// SPDX-FileCopyrightText: 2023 Klarälvdalens Datakonsult AB, a KDAB Group company <info@kdab.com>
+// SPDX-FileContributor: Andrew Hayzen <andrew.hayzen@kdab.com>
+//
+// SPDX-License-Identifier: MIT OR Apache-2.0
+
+mod main_object;
+
+// Ensure the symbols from the rlib dependencies end up
+// in the staticlib (if you use Rust symbols from these
+// crates in this crate, you can skip these `extern crate` statements).
+extern crate sub1;
+extern crate sub2;
```

Comment:
wonder if we should have a further `sub3` crate that uses types from sub1 and sub2 to check that is possible. Or is that not possible ? :thinking: Can be in a separate later branch though...

Reply:
That's sort of what the main crate does though, right?
Or do you mean use types in Rust?

Reply:
Right using the types, but lets wait for the pure virtual fun, as then we could have an abstract class and a concrete type in one crate. And then use the abstract version in the methods in another crate. Which would emulate some real world scenarios.